test/v2/controllers/V2MovementsControllerSpec.scala

```diff
@@ -406,7 +407,17 @@ class V2MovementsControllerSpec
             .thenAnswer {
               _ => EitherT.rightT(())
             }
-
+          when(
```

Comment:
Can we also add tests to verify that in some error scenarios, this is NOT called?